content/shmem_atomic_add.tex

```diff
@@ -58,6 +58,10 @@
     The \FUNC{shmem\_atomic\_add} routine performs an atomic add operation. It adds
     \VAR{value} to \dest{} on \ac{PE} \VAR{pe} and atomically updates the \dest{}
     without returning the value.
+    \newtext{
+    If the context handle \VAR{ctx} specifies an invalid context,
+    the behavior is undefined.
```

Comment:
Not quite sure why, but I think I prefer "If the context handle \VAR{ctx} does not correspond to a valid context, the behavior is undefined."  Seems like there are more paths to this handle corresponding to something invalid, than there are to something valid.